android/java/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/MarkerOptions.java

```diff
@@ -84,7 +89,7 @@ public boolean isVisible() {
     }
 
     public MarkerOptions position(LatLng position) {
-        ((Marker)annotation).position = position;
+        ((Marker)annotation).setPosition(position);
```

Comment:
This will cause breakages with JNI as described in https://github.com/mapbox/mapbox-gl-native/issues/2551.